processor_test.go

```diff
@@ -232,20 +232,6 @@ func Test_request_headers(t *testing.T) {
 	assert.Equal(t, "my-tenant", string(req.Header.Peek("X-Scope-OrgID")))
 }
 
-func Test_request_headers_with_prefix(t *testing.T) {
```

Comment:
This test is no longer worthwhile, since there is no logic in the `fillRequestHeaders()` method.